evalml/tests/problem_type_tests/test_problem_types.py

```diff
@@ -16,7 +16,7 @@ def correct_problem_types():
 
 
 def test_handle_string(correct_problem_types):
-    problem_types = ['regression', ProblemTypes.MULTICLASS, 'binary']
+    problem_types = ['regression', ProblemTypes.MULTICLASS, 'binary', ProblemTypes.TIME_SERIES_REGRESSION]
```

Comment:
how did you add `ProblemTypes.TIME_SERIES_REGRESSION` here without adding anything to `correct_problem_types`?

Reply:
Hm, I think we actually need to add to `correct_problem_types` for this test to be fully updated. Reason why it passes right now is because `zip(problem_types, correct_problem_types)` will only iterate when both lists still have items, so it only goes through `correct_problem_types` (3 elements)

Reply:
Good catch guys! I'll update this.

Reply:
Done!

Reply:
👏   great reviewing @bchen1116 @angela97lin !